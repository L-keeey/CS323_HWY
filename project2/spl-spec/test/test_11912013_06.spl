float addOne(int n)
{
  return n + 1;
}
int addOne(int n)
{
  return n + 1;
}
int test_11912013_4()
{
  int k = 4;
  int r = addOne(k);
  return r;
}