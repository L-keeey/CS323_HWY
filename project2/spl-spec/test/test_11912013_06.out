Error type 8 at Line 3: function return type and the declared type mismatch.
Error type 3 at Line 5: variable is redefined.
Error type 4 at Line 5: function should only be defined once.
Error type 5 at Line 12: the type of the operands at both sides of assignment operator should match.
